frontend_tests/node_tests/search_suggestion.js

```diff
@@ -211,6 +213,46 @@ global.stream_data.populate_stream_topics_for_tests({});
         "pm-with:ted@zulip.com",
     ];
     assert.deepEqual(suggestions.strings, expected);
+
+    // Make sure suggestions still work if preceding tokens
+    query = 'is:alerted sender:ted@zulip.com';
+    suggestions = search.get_suggestions(query);
+    expected = [
+        "is:alerted sender:ted@zulip.com",
+        "is:alerted is:private",
```

Comment:
I think `is:private` should probably not be in this list, what do you think?  Might be another follow-up issue.